colourgame.html

```diff
@@ -7,29 +7,40 @@
   </body>
 <script>
 var guess,
-  answer,
-  clue1,
-  clueguess;
+  	answer,
+  	response,
+  	color,
+  	number;
+
+number = Math.floor(Math.random() * 10);
```

Comment:
you could put red, blue, and green into an array and then select a random index from the array. 
it would be a bit more concise and a little easier to reason about. (you wouldn't need the if-then-else anymore)